multicore_runtime/wsgi_test.py

```diff
@@ -180,6 +202,46 @@ def test_static_file_mime_type(self):
       self.assertEqual(response.data, f.read())
     self.assertEqual(response.mimetype, 'application/fake_type')
 
+  def test_static_file_header(self):
+    response = self.client.get('/static_header')
+    self.assertEqual(response.status_code, httplib.OK)
+    with open(static_path('test_statics/favicon.ico')) as f:
+      self.assertEqual(response.data, f.read())
+    self.assertTrue(response.headers.has_key('X-Foo-Header'))
+    self.assertEqual(response.headers['X-Foo-Header'], 'foo')
+    self.assertTrue(response.headers.has_key('X-Bar-Header'))
+    self.assertEqual(response.headers['X-Bar-Header'], 'bar value')
+
+  def test_static_file_expires(self):
+    # Patching the datetime class
+    datetime.datetime = FakeDatetime
```

Comment:
This is cached globally, so patching the module in this manner makes the tests mutate global state. We can use the mock.patch context manager to do this more hermetically, in which case we also won't need FakeDatetime; a magicmock with the correct return value will do instead.